Домашнее задание -7

Nattalllia opened the pull request:
Вывод в интерфейс товары для животных

Nattalllia committed:
Домашнее задание -7

Вывод в интерфейс товары для животных

mdubik777 commented on script.js:
```diff
@@ -1,86 +1,117 @@
+}
+
+
+displayItems(items);
```
Наталья,  хорошо,  но лучше разделить на 2 функции, Всегда делить код по его назначению -  одна функция отвечает за отрисовку всех элементов,  вторая - за создания карточки. 
т.е в функции displayItems перебором для каждого элемента вызываем вторую функцию, которая вернет заполненную карточку (и записать её в переменную) и вставляем в контейнер

Nattalllia committed:
Домашнее задание -7

Разделение на две функции: 1)createItemCard(item) отвечает за создание карточки и 2) displayItems(items) отвечает за перебор и отображение товаров.
